lib/scanner/blockqueue.go

```diff
@@ -42,7 +42,7 @@ func newParallelHasher(dir string, blockSize, workers int, outbox, inbox chan pr
 
 // HashFile hashes the files and returns a list of blocks representing the file.
 func HashFile(path string, blockSize int, counter Counter, useWeakHashes bool) ([]protocol.BlockInfo, error) {
-	fd, err := os.Open(path)
+	fd, err := fs.DefaultFilesystem.Open(path)
```

Comment:
Hmm, this should probably bubble down all the way from the scanner if you want to declare that "scanner conversion is finished".